config/config.properties

```diff
@@ -97,6 +97,21 @@ pravegaservice.dataLog.impl.name=BOOKKEEPER
 # Default value: HDFS
 # pravegaservice.storage.impl.name=HDFS
 
+
+# Storage layout for Tier 2 storage.
+# Valid values:
+#   TABLE_BASED - Using TableStore.
+#   LEGACY      - Using header files.
```

Comment:
I am wondering too whether "LEGACY" is meaningful here. Maybe "ROLLING_STORAGE" (and somewhere put a link to the `RollingStorage.java` class on GitHub (branch 0.7 -since it will be gone from master at one point).

What do you think?

Reply:
Just changed this to ...
``` java
/**
 * Type of Storage metadata format.
 */
public enum StorageMetadataFormat {
    /**
     * Uses format that stores data in header files.
     */
    HEADER_BASED,

    /**
     * Uses format that stores data in table segments.
     */
    TABLE_BASED,
}

/**
 * Type of Storage layout to use.
 */
public enum StorageLayoutType {
    /**
     * Uses {@link io.pravega.segmentstore.storage.rolling.RollingStorage}.
     */
    ROLLING_STORAGE,

    /**
     * Uses {@link io.pravega.segmentstore.storage.chunklayer.ChunkedSegmentStorage} .
     */
    CHUNKED_STORAGE,
}
```